tests/resources/serializers/test_marcxml_serializer.py

```diff
@@ -0,0 +1,93 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2023 CERN
+#
+# Invenio-RDM-Records is free software; you can redistribute it and/or modify
+# it under the terms of the MIT License; see LICENSE file for more details.
+
+"""Resources serializers tests."""
+
+from collections import Iterable
+
+import pytest
+from dojson.contrib.marc21.utils import GroupableOrderedDict, create_record
+
+from invenio_rdm_records.resources.serializers.marcxml import MARCXMLSerializer
+
+
+@pytest.fixture(scope="function")
+def updated_minimal_record(minimal_record):
+    """Update fields (done after record create) for MARCXML serializer."""
+    minimal_record["access"]["status"] = "open"
+    for creator in minimal_record["metadata"]["creators"]:
+        name = creator["person_or_org"].get("name")
+        if not name:
+            creator["person_or_org"]["name"] = "Name"
+
+    return minimal_record
+
+
+@pytest.fixture(scope="function")
+def updated_full_record(full_record):
+    """Update fields (done after record create) for MARCXML serializer."""
+    full_record["access"]["status"] = "embargoed"
+
+    return full_record
+
+
+def test_marcxml_serializer_minimal_record(running_app, updated_minimal_record):
+    """Test serializer for MARCXML"""
+    serialized_record = {}
+    expected_data = {}
+
+    serializer = MARCXMLSerializer()
+    serialized_record = create_record(
+        serializer.serialize_object(updated_minimal_record)
+    )
+    expected_data = '<record xmlns="http://www.loc.gov/MARC21/slim"><datafield tag="245" ind1="a" ind2=" "><subfield>A Romans story</subfield></datafield><controlfield tag="540">info:eu-repo/semantics/openAccess</controlfield><datafield tag="260" ind1="c" ind2=" "><subfield>2020-06-01</subfield></datafield><datafield tag="100" ind1="a" ind2=" "><subfield>Name</subfield></datafield><datafield tag="100" ind1="a" ind2=" "><subfield>Troy Inc.</subfield></datafield><controlfield tag="901">info:eu-repo/semantic/other</controlfield><datafield tag="260" ind1="b" ind2=" "><subfield>Acme Inc</subfield></datafield></record>'
```

Comment:
The subfield tag is not correct. The subfields are missing the attributes (subfield codes). As an example from loc sandburg.

Reply:
Hmm you're right, I'll modify this thx